index.js

```diff
@@ -32,4 +33,5 @@ module.exports = {
   storage: storage.default,
   MemoryStore: MemoryStore.default,
   network: network.default,
+  Wallet: Wallet.default,
```

Comment:
I think there is a `wallet` here. So, we should rename it to something else. Maybe `simpleWallet`, or `HathorWallet`, or `WalletManager`. What do you think?

Reply:
I agree, it's on the description of the PR that I would like a better name for it. I like `HathorWallet`. Changed it